pkg/ctl/get/cluster.go

```diff
@@ -38,7 +38,7 @@ func getClusterCmd(cmd *cmdutils.Cmd) {
 	}
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
-		fs.StringVarP(&cfg.Metadata.Name, "name", "n", "", "EKS cluster name")
+		fs.StringVarP(&cfg.Metadata.Name, "cluster", "c", "", "EKS cluster name")
```

Comment:
eksctl follows the convention of using `--cluster` only for commands whose primary resource is not a cluster. While `--cluster` might be more consistent with other commands, unfortunately, this is a breaking change and we cannot go ahead with it.